gradle/published-java-module.gradle

```diff
@@ -10,70 +10,91 @@ apply from: rootProject.file( 'gradle/java-module.gradle' )
 apply from: rootProject.file( 'gradle/publishing-repos.gradle' )
 apply from: rootProject.file( 'gradle/publishing-pom.gradle' )
 
-
-
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 // Jar
 
 jar {
-	manifest = osgiManifest {
-		// GRADLE-1411: Even if we override Imports and Exports
-		// auto-generation with instructions, classesDir and classpath
-		// need to be here (temporarily).
-
-		if ( project.pluginManager.hasPlugin( 'groovy' ) ) {
-			classesDir = sourceSets.main.groovy.outputDir
-		}
-		else {
-			classesDir = sourceSets.main.output.classesDir
-		}
-		classpath = configurations.runtime
-
-
-		// Java 9 module name
-		instruction 'Automatic-Module-Name', project.java9ModuleName
-
-		// the OSGi metadata
-		symbolicName project.java9ModuleName
-		vendor 'Hibernate.org'
-		description project.description
-		docURL "http://www.hibernate.org/orm/${project.ormVersion.family}"
-
-		instruction 'Import-Package',
-					// Temporarily support JTA 1.1 -- Karaf and other frameworks still
-					// use it.  Without this, the plugin generates [1.2,2).
-					'javax.transaction;version="[1.1,2)"',
-					// Tell Gradle OSGi to still dynamically import the other packages.
-					// IMPORTANT: Do not include the * in the modules' .gradle files.
-					// If it exists more than once, the manifest will physically contain a *.
-					'*'
-
-		// Basic JAR manifest metadata
-		instruction 'Specification-Title', project.name
-		instruction 'Specification-Version', project.version
-		instruction 'Specification-Vendor', 'Hibernate.org'
-		instruction 'Implementation-Title', project.name
-		instruction 'Implementation-Version', project.version
-		instruction 'Implementation-Vendor', 'Hibernate.org'
-		instruction 'Implementation-Vendor-Id', 'org.hibernate'
-		instruction 'Implementation-Url', 'http://hibernate.org/orm'
-
-		instruction 'Hibernate-VersionFamily', project.ormVersion.family
-		instruction 'Hibernate-JpaVersion', project.jpaVersion.name
+	manifest {
+		attributes(
+				// Basic JAR manifest attributes
+				'Specification-Title': project.name,
+				'Specification-Version': project.version,
+				'Specification-Vendor': 'Hibernate.org',
+				'Implementation-Title': project.name,
+				'Implementation-Version': project.version,
+				'Implementation-Vendor': 'Hibernate.org',
+				'Implementation-Vendor-Id': 'org.hibernate',
+				'Implementation-Url': 'http://hibernate.org/orm',
+
+				// Java 9 module name
+				'Automatic-Module-Name': project.java9ModuleName,
+
+				// Hibernate-specific JAR manifest attributes
+				'Hibernate-VersionFamily': project.ormVersion.family,
+				'Hibernate-JpaVersion': project.jpaVersion.name,
+
+				// BND Plugin instructions (for OSGi):
+				'Bundle-Name': project.name,
+				'Bundle-SymbolicName': project.java9ModuleName,
+				'Bundle-Vendor': 'Hibernate.org',
+				'Bundle-DocURL': "http://www.hibernate.org/orm/${project.ormVersion.family}",
```

Comment:
Chrome says `http://www.hibernate.org/` and `https://www.hibernate.org/` are not secure.

Should the above be:

`'Bundle-DocURL': "https://hibernate.org/orm/${project.ormVersion.family}` ?

Reply:
Same as the other comments: if we change this, better do it on the `main` branch too (and maybe 5.4).